### DISCUSSION TITLE
진척사항 및 추후 일정 관리

### DISCUSSION THREAD

**Discussion by myrhymetree** [goalapa/spring-cacamuca#87] [Announcements]
- 주원
완료 : 카테고리 등록
개발 중  : 식품 목록 조회, 식품 등록, 식품 수정, 식품 삭제, 카테고리 삭제 기능

- 성준
완료 : 로그인/로그아웃, 아이디찾기/비밀번호찾기, 회원정보 변경, 회원탈퇴
개발 중 : 페이지 별 권한 처리, 관리자 페이지

- 수의
완료 : 신고 등록 기능 완료, 신고 목록 조회
개발 중 : 블랙리스트 관리

- 정인
완료 : 식품 등록 요청 저장 기능, 요청상세보기 기능 완료
개발 중 : 요청 수정, 요청 삭제

- 현석
완료 : 리뷰 작성 저장, 리뷰목록조회, 좋아요 저장
개발 중 : 좋아요 조회(실시간 좋아요 반영)

7/24일 자정 전까지 작업하고 이후 우선순위 선별
배포 전 개발 마감 : 7/27 자정
배포 : 7/27
PT 준비 작업 : 7/29~7/30

**Comment by myrhymetree** [comment]
@numerical43 @Dylan-SonJungin @bbbbooo @juunewon 
회의 내용 정리했습니다.

**Reply by Oleevia01** [reply]
고생하셨습니다! 추가 사항이 있어 수정하였습니다🙂

**Reply by myrhymetree** [reply]
@juunewon @numerical43 @Dylan-SonJungin @bbbbooo  실사용자들이 해보고 피드백 받고 디버깅 하려면  배포 하기로 한 예정일보더 앞당겨야 할 것 같은데 다들 어떻게 생각하시나요?

**Reply by silverpoodle** [reply]
언제까지로 생각하고 계세용??

**Reply by myrhymetree** [reply]
27일이 좋지 않을까 싶어요

**Reply by bbbbooo** [reply]
괜찮습니다~

**Reply by numerical43** [reply]
넵 27일 좋습니다!

**Reply by Oleevia01** [reply]
27일 확인했습니다!
